opentelemetry-api/src/opentelemetry/context/async_context.py

```diff
@@ -12,32 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import typing  # pylint: disable=unused-import
-from contextvars import ContextVar
+try:
+    from contextvars import ContextVar
+except ImportError:
+    pass
+else:
+    import typing  # pylint: disable=unused-import
+    from . import base_context
 
-from . import base_context
+    class AsyncRuntimeContext(base_context.BaseRuntimeContext):
+        class Slot(base_context.BaseRuntimeContext.Slot):
+            def __init__(self, name: str, default: "object"):
```

Comment:
couldn't the default argument just be the object object, vs the string "object"?

Reply:
Yes, it could. Updated.